ee-9/pom.xml

```diff
@@ -71,6 +71,7 @@
         <!--<version.jakarta.xml.bind.jakarta-xml-bind-api>3.0.0</version.jakarta.xml.bind.jakarta-xml-bind-api>-->
         <version.org.apache.myfaces.core>3.0.1</version.org.apache.myfaces.core>
         <version.org.eclipse.yasson>2.0.1</version.org.eclipse.yasson>
+        <version.org.eclipselink.version>3.0.2</version.org.eclipselink.version>
```

Comment:
This property has no effect. To change the version used in the ee-9 modules you'd need to add an entry for this artifact into the dependencyManagement section of this pom.